schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
      date_of_birth      date not null,
      escape_attempts    int not null,
      neutered           boolean not null,
-     weight_kg          decimal(10, 2)
+     weight_kg          decimal(10, 2),
+     species             varchar
```

Comment:
- [ ] You were expected to add a new column called species with a string type using the `ALTER TABLE` command like so 👇🏾 

```sql
ALTER TABLE animals ADD species varchar(255);
```

Please do this instead

Reply:
Sure thing, I have used the similar SQL but made mistake while adding it in to schema.sql. Thank you so much for clarity! @bobb-Rob 🤙